RocketChat/Rocket.Chat
# [IMPROVE] Replace Site_Name and Site_Url config name

## Proposed changes (including videos or screenshots)


![image](https://user-images.githubusercontent.com/27704687/119159865-12b44c80-ba2e-11eb-8998-b10f913953fa.png)
![image](https://user-images.githubusercontent.com/27704687/119159894-1a73f100-ba2e-11eb-827c-6518f4237c12.png)




## Issue(s)


## Steps to test or reproduce

- Go to Admin > General 
- Check the new labels Workspace URL and Workspace Name instead of Site URL and Site Name

## Further comments
